.github/workflows/translation.yml

```
@@ -1,47 +1,77 @@
 name: Translation
 
 on:
-   workflow_run:
-     workflows:
-     - Test
-     branches:
-     - main
-     types:
-     - completed
+  workflow_run:
+    workflows:
+    - Test
+    branches:
+    - main
+    types:
+    - completed
+
+env:
+  I18N_BRANCH: translation/source
 
 jobs:
-   build:
-     runs-on: ubuntu-latest
-
-     steps:
-       - name: Grab the repo src
-         uses: actions/checkout@v2
-
-       - name: Set up Python
-         uses: actions/setup-python@v2
-         with:
-           python-version: 3.9
-
-       - name: Install Python tooling
-         run: python -m pip install --upgrade nox virtualenv
-
-       - name: Generate a fresh POT file out of RST documents
-         run: python -m nox -s translation
-
-       - name: Commit the POT file to Git
-         run: |
-           git config --local user.email "github-actions[bot]@users.noreply.github.com"
-           git config --local user.name "github-actions[bot]"
-           git_hash=$(git rev-parse --short "${GITHUB_SHA}")
-           git commit -m "Update messages.pot as of version ${git_hash}" locales/messages.pot
-
-       - name: Check if any sources have changed since the last update
-         if: failure()
-         run: echo "There are no changes to the RST sources since the last update. Nothing to do."
-
-       - name: >-
-           Push the updated POT file back to
-           ${{ github.repository }}@${{ github.event.repository.default_branch }}
-           on GitHub
-         run: |
-           git push --atomic origin HEAD:${{ github.event.repository.default_branch }}
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Grab the repo src
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 0  # To reach the common commit
+    - name: Setup git user as [bot]
```

Consider the verb "Set up", as "setup" is technically noun #nitpicky

---

Yeah, you're right. Currently, there's a more serious problem with this: Weblate doesn't work like Pradyun and I expected and now we need to think about how to address that...
```suggestion
    - name: Set up git user as [bot]
```